jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -2463,6 +2463,11 @@ properties:
             type: string
             fits_keyword: S_TWKREG
             blend_table: True
+          undersampling_correction:
+            title: Undersampling Correction
+            type: string
+            fits_keyword: S_UNDCOR
```

Comment:
I think I'd rather see the available characters get allocated towards more of the actual step name, rather than including "COR" (for correction) (most steps are corrections of some sort). How about `S_UNDSMP`?

Reply:
Done